src/risk_distributions/risk_distributions.py

```diff
@@ -524,14 +490,17 @@ def cdf(self, x: Union[pd.Series, np.ndarray, float, int]) -> Union[pd.Series, n
         computable = weights[(weights.sum(axis=1) != 0) & ~np.isnan(x)].index
 
         c = pd.Series(np.nan, index=x.index)
-
         c.loc[computable] = 0
 
+        distribution_map = pd.Series([np.random.choice(weights.columns, p=weights.iloc[0].values) for _ in x.index], index=x.index)
```

Comment:
Three things:
- In scientific computing you pretty much never ever ever want to use unseeded randomness
- When dealing with distributions, `cdf`, `pdf`, and `ppf` are expected to be deterministic functions of their inputs.  You don't want randomness here at all.
- When working with random numbers in our simulations, all random numbers should be bound to `vivarium`'s randomness system (which is why you should provide a propensity for selecting distributions as an argument).